### DISCUSSION TITLE
v2.0.0-beta1

### DISCUSSION THREAD

**Discussion by mosajjal** [mosajjal/sniproxy#45] [Announcements]
https://github.com/mosajjal/sniproxy/releases/tag/v2.0.0-beta1

just like all the alpha releases, I encourage testing this and provide early feedback :)

notable changes from the alpha version:
- logging is overhauled and looks much cleaner 
- DNS has better performance and works under all conditions

getting closer to a code freeze and a stable release!

**Comment by holygrolli** [comment]
are you sure the version was built right? I downloaded https://github.com/mosajjal/sniproxy/releases/download/v2.0.0-beta1/sniproxy-v2.0.0-beta1-linux-amd64.tar.gz and the  `--help` shows me command line arguments from v1 and I cannot pass my yaml config 🤔

**Comment by mosajjal** [comment]
sorry. it's a GH CLI weirdness :-/ beta2 should be ok. let's move there (#46).
